image_lists/hrrr_subset.yml

```diff
@@ -83,6 +83,7 @@ hourly:
     rh:
       - 2m
       - 850mb
+      - mean
```

Comment:
This should also show up in any of the other image lists that are relevant...rap, hrrr, etc.

Reply:
I'll add it in rap.yml and it should be present for most yml files.